File: ws-spark-java/spark-java/src/main/java/Ex01ReadTextFile.java
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;

/**
 * Read a text file as an RDD of Strings
 */
public class Ex01ReadTextFile {
    public static void main(String[] args){

        SparkConf conf = new SparkConf();
        conf.setAppName("Application Ex01ReadTextFile");
        conf.setMaster("local");
        JavaSparkContext sc = new JavaSparkContext(conf);
        JavaRDD<String> rdd =
                sc.textFile(ModuleEnv.DATA_ROOT +"\\ml-latest-small\\movies.csv");

        rdd.take(10).forEach(line -> System.out.println(line));
    }
}

File: ws-spark-scala/spark-scala/src/main/scala/Ex01ReadTextFile.scala
import org.apache.spark.{SparkConf, SparkContext}

/**
  * Read a text file as an RDD of Strings
  */
object Ex01ReadTextFile extends App{
  val conf = new SparkConf()
  conf.setAppName("Spark-Hello-World")
  conf.setMaster("local")
  val sc = new SparkContext(conf);
  val rdd = sc.textFile(ModuleEnv.DATA_ROOT+ "\\ml-latest-small\\movies.csv")

  rdd.take(10).foreach(println)
}
